include/ofi_hmem.h

```diff
@@ -132,6 +141,18 @@ static inline int ofi_hmem_no_close_handle(void *ipc_ptr)
 	return -FI_ENOSYS;
 }
 
+static inline
+int ofi_hmem_no_dev_register(struct fi_mr_attr *mr_attr, uint64_t *handle)
+{
+	return -FI_ENOSYS;
+}
+
+static inline
+int ofi_hmem_no_dev_deregister(uint64_t handle)
+{
+	return -FI_ENOSYS;
+}
```

Comment:
If dev registration is not required, can this return success instead?

I'd like this change separated out (hold off for now), but mostly see if it needs to be aligned with PR #6176, which adds a host register call...

Reply:
Sure, return success here make sense.

I will move the abstraction of ```dev_register``` and ```dev_deregister``` to a separate PR then